LabBuilder/LabBuilder.psm1

```diff
@@ -2774,7 +2889,8 @@ Add-Content ``
     $SetupCompleteCmd = @"
 @echo SetupComplete.cmd Script Started... >> %SYSTEMROOT%\Setup\Scripts\SetupComplete.log
 $SetupCompleteCmd
-powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`"
+Timeout 30
+powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`" `r
```

Comment:
I have no explanation - I tested several combinations without the timeout and the CR added and this was the least modification that worked.